packages/node-resolve/README.md

```diff
@@ -118,10 +118,12 @@ DEPRECATED: use "resolveOnly" instead
 ### `preferBuiltins`
 
 Type: `Boolean`<br>
-Default: `true`
+Default: unset
 
 If `true`, the plugin will prefer built-in modules (e.g. `fs`, `path`). If `false`, the plugin will look for locally installed modules of the same name.
 
+If unset the plugin will first look for modules of the same name, and then fall back to built-in modules.
```

Comment:
If we keep the warning about `unset` that still appears to be in the code, I guess this should be mentioned here, otherwise people think they can get away with not configuring this option when they use imports like `fs`. Alternatively, we could allow this as a valid "third"  option, e.g. `"auto"`, where you want to override some builtins with local alternatives but keep others. But I am not sure I can imagine a scenario where people would want this.

Reply:
The plugin behaves a bit differently when it's unset. I was confused by this because I though it would default to either the `true` or `false` behaviour, but it's essentially a third option, so this was to try and make that clear.

The way these 2 variables are used makes the third behaviour
```js
  const isPreferBuiltinsSet = options.preferBuiltins === true || options.preferBuiltins === false;
  const preferBuiltins = isPreferBuiltinsSet ? options.preferBuiltins : true;
```

I'd be happy for it to be an explicit option. Are you suggesting that `unset` should become `auto`? or something else?

Reply:
Looking at the code again now I'm not actually sure if the line I added here is correct.  It's quite confusing to follow through the logic

Reply:
Your example looks very much like the `default` for `preferBuiltins` would be `true`, just with a warning about it.

Reply:
Yeh removing this comment. Opened another PR to simplify how we handle `preferBuiltins` a bit

https://github.com/rollup/plugins/pull/637